docker-compose.yaml

```diff
@@ -104,8 +104,8 @@ services:
     image: ipfs/kubo:latest
     ports:
       - 4001:4001
-      - 127.0.0.1:5001:5001
-      - 127.0.0.1:8080:8080
+      - 5001:5001
+      - 8080:8080
```

Comment:
Should probably leave `127.0.0.1` in there; it's recommended not to expose those ports to the outside. (Maybe 8080 if you're running a public gateway, but not 5001)
https://docs.ipfs.tech/install/run-ipfs-inside-docker/#set-up

Reply:
(`127.0.0.1` should be the default; maybe add some comments about changing/removing it if your local setup involves multiple dev machines). Remember this compose file is only intended for local development, not production.